data/human/jobs.txt

```diff
@@ -4167,3 +4167,947 @@ mission "Republic Navy Advisory System [wildfire firefighters]"
+	npc
+		government "Bounty Hunter"
+		personality heroic unconstrained nemesis plunders harvests waiting
+		fleet "Small Core Pirates"
+		fleet "Marauder fleet IV"
+		system
+			distance 1 2
+	npc
+		government "Syndicate"
+		personality heroic unconstrained nemesis waiting
+		fleet "Small Syndicate"
+		fleet "Large Syndicate" 2
+		system
+			distance 3 5
+
+mission "Transport recruiters [0]"
+	name "Recruiter and supplies to <planet>"
+	job
+	repeat
+	description `Transport a <origin> recruiter and <tons> of recruitment supplies to <planet> by <date> for <payment>.`
+	passengers 1
+	cargo "recruiting supplies" 2 5 0.8
+	deadline 5 3
+	to offer
+		random < 40
+	source "New Tortuga"
+	destination
+		government "Republic" "Pirate" "Free Worlds" "Neutral" "Independent"
+		not attributes uninhabited
+		attributes farming mining research factory urban
+	on complete
+		payment 7000 170
+		dialog phrase "generic passenger dropoff payment"
+	on visit
+		dialog phrase "generic passenger on visit"
+
+mission "Transport recruiters [1]"
+	name "Recruiters and supplies to <planet>"
+	job
+	repeat
+	description `Transport <bunks> <origin> recruiters and <tons> of recruitment supplies to <planet> by <date> for <payment>.`
+	passengers 2 5 0.8
+	cargo 4 12 0.8
+	deadline 5 3
+	to offer
+		random < 40
+	source "New Tortuga"
+	destination
+		government "Republic" "Pirate" "Free Worlds" "Neutral" "Independent"
+		not attributes uninhabited
+		attributes farming mining research factory urban
+	on complete
+		payment 7000 170
+		dialog phrase "generic passenger dropoff payment"
+	on visit
+		dialog phrase "generic passenger on visit"
+
+mission "Transport recruiters [2]"
+	name "Recruiters and supplies to <planet>"
+	job
+	repeat
+	description `Transport a team of <bunks> <origin> recruiters and <tons> of recruitment supplies to <planet> by <date> for <payment>.`
+	passengers 7 20 0.8
+	cargo 12 40 0.8
+	deadline 5 3
+	to offer
+		random < 40
+		"cargo space" > 10
+		"passenger space" > 5
+	source "New Tortuga"
+	destination
+		government "Republic" "Pirate" "Free Worlds" "Neutral" "Independent"
+		not attributes uninhabited
+		attributes farming mining research factory urban
+	on complete
+		payment 7000 170
+		dialog phrase "generic passenger dropoff payment"
+	on visit
+		dialog phrase "generic passenger on visit"
+
+mission "Retrieve recruiters [0]"
+	name "Return a recruiter home to <planet>"
+	job
+	repeat
+	description `Pick up a recruiter on <stopovers> and fly them home to <planet> by <date> for <payment>.`
+	passengers 1
+	deadline 5 3
+	to offer
+		random < 40
+	source "New Tortuga"
+	stopover
+		government "Republic" "Pirate" "Free Worlds" "Neutral" "Independent"
+		attributes farming mining research factory urban
+		not attributes uninhabited
+	on complete
+		payment 7000 170
+		dialog phrase "generic passenger dropoff payment"
+	on visit
+		dialog phrase "generic passenger on visit"
+	on stopover
+		dialog phrase "generic passenger stopover pickup"
```

Comment:
```suggestion
		dialog `Your <passengers> boards your ship, ready to go to <planet>.`
```

Reply:
Alternatively, could do 
```suggestion
		dialog `Your one passenger boards your ship, ready to go to <planet>.`
```
for clarity to future reviewers/people reading the files (because passengers seems to be plural)

Reply:
Right!  Or better...

```suggestion
		dialog `The recruiter boards your ship, ready to go to <planet>.`
```